dataAccessObjects.py

```diff
@@ -25,8 +25,8 @@ def queryOneValue(self,sql):
 class DataManipulateDao(DefaultDao):
     #TODO handle if derived class not assign dataName
     def markExpired(self,targetId):
-        sql = 'UPDATE {dataName}_data SET {dataName}_is_expire=1 WHERE {dataName}_id="{targetId}"'.format(
-                dataName=self.dataName,targetId=targetId)
+        sql = 'UPDATE {tableName}_data SET {dataName}_is_expire=1 WHERE {dataName}_id="{targetId}"'.format(
```

Comment:
`{tableName}_data` should be `{tableName}` because tableName already contain "_data"